### PR TITLE
Add page model tests to dashboard

### PR DESCRIPTION
**Pull request by echappen**
Proof of concept of some basic tests for the Get and Request methods on a page model. Shows how to mock HttpContext so objects like `Request.Query` and `Request.Form` can be manipulated in the tests.

Nothing super specific I'm looking for feedback on here, just interested to hear anyone's general thoughts.

### CODE REVIEW

File: dashboard/tests/Piipan.Dashboard.Tests/ParticipantUploadsModelTests.cs [hunk 1]
```diff
@@ -41,6 +49,90 @@ public void BeforeOnGetAsync_initializesParticipantUploadResults()
             Assert.IsType<List<ParticipantUpload>>(pageModel.ParticipantUploadResults);
         }
 
-        // Add more here
+        // sets participant uploads after Get request
+        [Fact]
+        public async void AfterOnGetAsync_setsParticipantUploadResults()
+        {
+            // setup env
+            Environment.SetEnvironmentVariable("MetricsApiUri", "http://example.com");
```
Comment (ryanhofdotgov) [inline dashboard/tests/Piipan.Dashboard.Tests/ParticipantUploadsModelTests.cs:57]: suggestion/nitpick: define the variable name as a public constant in the class/package under test

Comment (ryanhofdotgov) [reply dashboard/tests/Piipan.Dashboard.Tests/ParticipantUploadsModelTests.cs:57]: Thanks for forging ahead with this – looks like a great approach we will incorporate pervasively across the 2 web apps.

Comment (echappen) [reply dashboard/tests/Piipan.Dashboard.Tests/ParticipantUploadsModelTests.cs]: Updated [here](https://github.com/18F/piipan/pull/427/commits/ab2914571bfb151d2e58e88508c0a6d2a4136378), is this what you were thinking?

Comment (ryanhofdotgov) [reply dashboard/tests/Piipan.Dashboard.Tests/ParticipantUploadsModelTests.cs]: Yeah, thanks 👍

**Review by ryanhofdotgov — APPROVED** [review]
Wasn't sure if there was more to come, but as far as what you have, 👍 from me

**Pull request merged**